tests/system/ControllerTest.php

```diff
@@ -75,7 +75,7 @@ public function testConstructorHTTPS()
         $original = $_SERVER;
         $_SERVER  = ['HTTPS' => 'on'];
         // make sure we can instantiate one
-        $this->controller         = new class () extends Controller {
+        $this->controller = new class () extends Controller {
```

Comment:
Yay! CS Fixer has been weird about anonymous class assign spacing ever since you introduced me. Happy to see this working better.